apps/rotm/index.js

```diff
@@ -48,7 +49,8 @@ module.exports = {
     },
     '/check-your-report': {
       prereqs: ['/image'],
-      behaviours: [require('hof-behaviour-summary-page'), 'complete', caseworkerEmailer, checkReportBackLink],
+      behaviours: [require('hof-behaviour-summary-page'), 'complete',
+        caseworkerEmailer, checkReportBackLink, removeImage],
```

Comment:
If you want to put it over two lines, it read better with each behaviour on a new line

Reply:
do you mean like this
```
behaviours: [
     require('hof-behaviour-summary-page'), 
     'complete',
     caseworkerEmailer, 
      checkReportBackLink, 
      removeImage
],
```

Reply:
yep, something like that.

Reply:
ok updated